spec/features/publish/courses/editing_course_information_spec.rb

```diff
@@ -51,6 +51,18 @@
             how_school_placements_work: "")
     end
 
+    scenario "the course does not display its own name in the copy list" do
+      publish_course_information_page.load(
+        provider_code: provider.provider_code, recruitment_cycle_year: provider.recruitment_cycle_year, course_code: course.course_code,
+      )
+
+      list_options = publish_course_information_page.copy_content.copy_options
+      expect(Course.count).to eq 3
+      expect(list_options.size).to eq 3
+      expect(list_options.shift).to eq("Pick a course")
+      expect(list_options.any? { |x| x[@course.name] }).to be_falsey
+    end
+
```

Comment:
This test passes on master. 

Can we maybe have a test that lists out all the courses and lists out all the course options in the list, demonstrating that one of the courses is not in the list?

Reply:
I think this is what it does, there are 3 courses, and 3 text options, the first one being 'pick a course', the remaining 2 being course2 and course3. The course name is not in the list `expect(list_options.any? { |x| x[@course.name] }).to be_falsey` , or did you mean something else?